base/io.jl

```diff
@@ -329,6 +367,19 @@ htol(x)
     isreadonly(stream) -> Bool
 
 Determine whether a stream is read-only.
+
+# Examples
```

Comment:
And here; Perhaps I was completely wrong when I thought that most signatures used `io` :smile: But I still think it is nice to be consistent :slightly_smiling_face: